docs/product-user-guide/src/main/asciidoc/proc-business-central-settings-artifacts.adoc

```diff
@@ -0,0 +1,4 @@
+[[business_central_settings_artifacts]]
+= Artifacts
+
+In the Artifacts perspective you can manage the content of the Maven repository. The artifacts that you build from your projects are listed here, along with their `GroupId`, `ArtifactId`, and `Version` (GAV). You can view the contents an artifact, as well as upload and download artifacts.
```

Comment:
Missing comma afte r"perspective": "In the Artifacts perspective. you can manage the..."